draft-kuehlewind-shmoo-remote-fee.md

```diff
@@ -120,6 +120,23 @@ It is not in scope for this document or the shmoo working group to make suggesti
 the IETF's overall funding model. This is the responsibility of the LLC Board taking agreed principles
 like the one proposed in this document into account.
 
+# Considerations on Use and Misuse of a Free Participation Option
+
+This document does not specify hard any requirements on when to use or not use the free option. The purpose of
+the free option is to enable everybody who is interested in participation to join meetings without the meeting
+fee imposing a financial barrier. These cases cannot be limited to a certain group, like students or "self-founded"
+participants, nor to any specific other restricts, like the number of meeting or previous level of involvement.
+The purpose is simply to maximise participation without barriers in order to make the standards process as open as possible.
+
+It is expected that participants who have financial support to use the regular registration option 
```

Comment:
s/It is expected that participants/Participants/  remove passive voice. :)

Reply:
This not passive voice. Your suggestion breaks the sentence...

Reply:
okay.